Code/src/main/java/nl/utwente/viskell/ui/components/Lane.java

```diff
@@ -227,7 +227,7 @@ public Node asNode() {
     
     @Override
     protected double computePrefWidth(double height) {
-        guardSpace.setPrefWidth(resizer.getBoundsInParent().getMaxX());
+        guardSpace.setPrefWidth(resizer.getBoundsInParent().getMaxX()-5);
```

Comment:
Don't question the magic numbers! 
I hate them as much as the next person, but @comonoidial stated it is 5 considering all the other integer constants currently in the project.